source/precalculus/source/meta/identities.ptx

```diff
@@ -10,6 +10,65 @@
                 <li><m>1+\cot^2(\theta)=\csc^2(\theta)</m></li>
             </ul>
         </statement>
+      </identity>
+
+      <identity>
+        <title>Sum and Difference Formulas</title>
+        <statement>
+            <ul>
+                <li><m>\sin \left(\alpha+\beta\right) = \sin\alpha\cos\beta+\cos\alpha\sin\beta</m></li>
+                <li><m>\sin \left(\alpha-\beta\right) = \sin\alpha\cos\beta-\cos\alpha\sin\beta</m></li>
+                <li><m>\cos \left(\alpha+\beta\right) = \cos\alpha\cos\beta-\sin\alpha\sin\beta</m></li>
+                <li><m>\cos \left(\alpha-\beta\right) = \cos\alpha\cos\beta+\sin\alpha\sin\beta</m></li>
+                <li><m>\tan \left(\alpha+\beta\right) = \frac{\tan\alpha+\tan\beta}{1-\tan\alpha\tan\beta}</m></li>
+                <li><m>\tan \left(\alpha-\beta\right) = \frac{\tan\alpha-\tan\beta}{1+\tan\alpha\tan\beta}</m></li>
+              </ul>
+        </statement>
+        </identity>
+        
+        <identity>
+        <title>Double Angle Formulas</title>
+        <statement>
+            <ul>
+                <li><m>\sin(2\theta)=2\sin\theta\cos\theta</m></li>
+                <li><m>\cos(2\theta)=\cos^2\theta-\sin^2\theta=1-2\sin^2\theta=2\cos^2\theta-1</m></li>
+                <li><m>\tan(2\theta)=\frac{2\tan\theta}{1-\tan^2\theta}</m></li>
+            </ul>
+        </statement>
+        </identity>
+        
+      <identity>
+        <title>Power Reduction Formulas</title>
+        <statement>
+            <ul>
+                <li><m>\sin^2\theta=\frac{1-\cos2\theta}{2}</m></li>
+                <li><m>\cos^2\theta=\frac{1+\cos2\theta}{2}</m></li>
+                <li><m>\tan^2\theta=\frac{1-\cos2\theta}{1+\cos\theta}</m></li>
```

Comment:
```suggestion
                <li><m>\tan^2\theta=\frac{1-\cos2\theta}{1+\cos2\theta}</m></li>
```

Double check me on this one, but it should be $2\theta$ in the denominator as well, right?

Reply:
yes! I'll fix it now!